src/System.Private.ServiceModel/tests/Scenarios/Binding/Http/NetHttpsBindingTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+public static class Binding_Http_NetHttpsBindingTests
+{
+    [Fact]
+    [OuterLoop]
+    public static void StandardUsage_NetHttps_Echo_RoundTrips_String()
```

Comment:
One of the other tests was using the binding to create the factory, so I was thinking of this particular style as being the standard way of creating a binding, factory and channel. But I guess that is pretty subjective. I'll update the name.